### PR TITLE
New package: poco-1.7.8

### PR DESCRIPTION
**Pull request by juliendehos**

### CODE REVIEW

File: srcpkgs/poco-devel/template [hunk 1]
```diff
@@ -0,0 +1 @@
+../poco/template
```
Comment (ebfe) [inline srcpkgs/poco-devel/template]: poco-devel should be a symlink to poco

Comment (juliendehos) [inline srcpkgs/poco-devel/template]: ok, fixed

File: srcpkgs/poco/template [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+post_install() {
+	vlicense LICENSE
+}
+
+poco_package() {
+	short_desc+=" - runtime libraries"
+	pkg_install() {
+		vmove usr/lib/*.so.*
+	}
+}
```
Comment (Vaelatern) [inline srcpkgs/poco/template]: this entire `poco_package` is useless and xbps-src doesn't even try to use it. Please remove it.

File: srcpkgs/poco/template [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+poco-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/*.so*
```
Comment (Vaelatern) [inline srcpkgs/poco/template]: `vmove usr/lib/*.so`

Comment (juliendehos) [inline srcpkgs/poco/template]: ok, fixed (vmove and poco_package)

**Pull request closed without merging**